src/listener/base-listener.js

```diff
@@ -58,6 +61,22 @@ class Listener {
     }
   }
 
+  pagination(msg, response, text) {
+    if (text) {
+      this.transform.text(msg, text);
+    }
+
+    this.makePagination(paginationJump, response.members, msg, response);
+  }
+
+  makePagination(jump, members, msg, response) {
+    if (jump < members.length + paginationJump) {
+      response.members = members.slice(jump - paginationJump, jump);
+      this.transform.send(msg, response);
+      this.makePagination(jump + paginationJump, members, msg, response);
+    }
+  }
+
```

Comment:
I like this code! It would be great if we could add some tests for this in the future.

Reply:
Thanks! yes, I'll do that.